test/e2e/helpers.js

```diff
@@ -163,9 +163,7 @@ async function withFixtures(options, testSuite) {
     staticServerOptions,
     title,
     ignoredConsoleErrors = [],
-    dappPath = undefined,
     disableServerMochaToBackground = false,
-    dappPaths,
```

Comment:
ℹ️  this is the main change of this PR: consolidate `dapp`, `dappPath`, `dappPaths` and `dappOptions` into just `dappOptions`